lib/ecto/adapters/sqlite3.ex

```diff
@@ -415,6 +415,11 @@ defmodule Ecto.Adapters.SQLite3 do
     [type, &Codec.time_encode/1]
   end
 
+  @impl Ecto.Adapter
+  def dumpers(:boolean, type) do
+    [type, &Codec.bool_encode/1]
+  end
+
```

Comment:
Let's remove this. There is already a function header that matches this.

```suggestion
```

Reply:
Will amend and do a squash.

Reply:
That's fine